driver.php

```diff
@@ -203,14 +203,17 @@
 </p>
 <pre>
 <b>$</b> gcc -v hello.c
+COLLECT_GCC_OPTIONS=...
+ /usr/lib/gcc/aarch64-linux-gnu/11/cc1 hello.c ... -o /tmp/ccCKUy4c.s
 COLLECT_GCC_OPTIONS=...
  as -v -EL -mabi=lp64 -o /tmp/ccUkKMFH.o /tmp/ccCKUy4c.s
-COMPILER_PATH==...
+COMPILER_PATH=...
+LIBRARY_PATH=...
 COLLECT_GCC_OPTIONS=...
- /usr/lib/gcc/aarch64-linux-gnu/11/collect2 /tmp/ccIPliNQ.o
+ /usr/lib/gcc/aarch64-linux-gnu/11/collect2 ... /tmp/ccUkKMFH.o ...
```

Comment:
The intro explains that outputs are modified for typesetting purposes. I assume you wanted to add this ellipsis to explain that there is more stuff here but I find it more visually confusing.

Reply:
Agreed.